joint_1:
    controller:
        package: mx_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_1
    joint_speed: 0.2
    motor:
        id: 1
        init: 0
        min: 0
        max: 4096
joint_2:
    controller:
        package: mx_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_2
    joint_speed: 0.2
    motor:
        id: 2
        init: 0
        min: -28672
        max: 28672
joint_3:
    controller:
        package: mx_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_3
    joint_speed: 0.3
    motor:
        id: 3
        init: 0
        min: -28672
        max: 28672
joint_4:
    controller:
        package: mx_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_4
    joint_speed: 0.2
    motor:
        id: 4
        init: 0
        min: -28672
        max: 28672
joint_5:
    controller:
        package: mx_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_5
    joint_speed: 0.2
    motor:
        id: 5
        init: 0
        min: -28672
        max: 28672
joint_6:
    controller:
        package: mx_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint_6
    joint_speed: 0.2
    motor:
        id: 6
        init: 2048
        min: 0
        max: 4096
